pkgs/unitech/README.md

```diff
@@ -0,0 +1,15 @@
+# unitech_scanner
+
+A new flutter plugin project.
+
+## Getting Started
+
+This project is a starting point for a Flutter
+[plug-in package](https://flutter.dev/developing-packages/),
+a specialized package that includes platform-specific implementation code for
+Android and/or iOS.
+
+For help getting started with Flutter, view our
+[online documentation](https://flutter.dev/docs), which offers tutorials,
+samples, guidance on mobile development, and a full API reference.
+
```

Comment:
Also add a simple example. We should also specify that combined_barcode_scanner is required as well
```
BarcodeScannerWidget(
        controller: _controller,
        onScan: (code) {
          if (kDebugMode) {
            print("GOT BARCODE =========== ${code.code}");
          }
        },
        configuration: const ScannerConfiguration(
          enableFormats: {BarcodeFormat.qr},
        ),
        scanners: [UnitechBarcodeScanner()],
      ),
```